maestro-client/src/main/java/maestro/Maestro.kt

```diff
@@ -126,6 +126,24 @@ class Maestro(private val driver: Driver) : AutoCloseable {
         waitForAppToSettle()
     }
 
+    fun swipe(startRelative: String, endRelative: String, durationMs: Long) {
```

Comment:
I think this should be outside of `Maestro` class as Maestro is providing a strict/deterministic API. This can be done in Orchestra instead